src/content/docs/cypher/data-types/list.md

```diff
@@ -75,3 +77,15 @@ Output:
 | 5 |
 -----
 ```
+
+## ARRAY
```

Comment:
@andyfengHKU Could you take a quick look at the new docs below for the `ARRAY` type? I didn't include any examples of how we can create `ARRAY` in Cypher (I'm not even sure we can). Does the explanation below make sense?